config/settings/wdb.q

```diff
@@ -58,9 +62,8 @@ tpcheckcycles:0W
 
 // Server connection details
 \d .servers
-CONNECTIONS:`hdb`tickerplant`rdb`gateway`sort                                               // list of connections to make at start up
```

Comment:
why is this removed?

Reply:
It was simply just overwritten in processes/wdb.q anyways.
Looks like it created the list of connections based on process types later anyways.
Not sure then if it makes sense to make it as a setting.

Reply:
Okay I think I see now why this is there. Not the fact that those types are listed but rather that is makes it possible for the users to add extra stuff here.
Added back!